### DISCUSSION TITLE
Markdown field not rendering Markdown code

### DISCUSSION THREAD

**Discussion by ronvdc** [statamic/cms#4351] [Troubleshooting]
I have a blueprint with a Markdown field that has also enabled Antlers. However, when I view the site, that field on the page doesn't show rendered Markdown, but the actual Markdown code. Also, even though the field is set to process Antlers in the blueprint definition ("antlers: true"), no Antler code is evaluated.
I can't seem to figure out why this is. Am I missing something obvious here?

Thanks!
--Ron

**Comment by jackmcdade** [comment]
Are you sure you're on the right page? Can you post your blueprint and template code?
